plugins/ommysql/ommysql.c

```diff
@@ -533,6 +544,8 @@ CODEmodInit_QueryRegCFSLineHdlr
 	STD_LOADABLE_MODULE_ID));
 	CHKiRet(omsdRegCFSLineHdlr((uchar *)"resetconfigvariables", 1, eCmdHdlrCustomHandler, resetConfigVariables,
 	NULL, STD_LOADABLE_MODULE_ID));
+	CHKiRet(omsdRegCFSLineHdlr((uchar *)"ommysqlsocket", 0, eCmdHdlrGetWord, NULL, &cs.pszMySQLSocket,
```

Comment:
we do not add new obsolete legacy style config directives. So please remove them. In the light of that, I would also **not** store the socket name inside the legacy ``cs`` structure.